test/extended/util/test.go

```diff
@@ -476,6 +478,13 @@ var (
 			`\[sig-storage\] In-tree Volumes \[Driver: azure\] \[Testpattern: Pre-provisioned PV \(ext4\)\] volumes should be mountable`,
 			`\[sig-storage\] In-tree Volumes \[Driver: azure\] \[Testpattern: Pre-provisioned PV \(filesystem volmode\)] volumeMode should create sc, pod, pv, and pvc, read/write to the pv, and delete all created resources`,
 		},
+		"[Skipped:gce]": {
+			// Requires creation of a different compute instance in a different zone and is not compatible with volumeBindingMode of WaitForFirstConsumer which we use in 4.x
+			`\[sig-scheduling\] Multi-AZ Cluster Volumes \[sig-storage\] should only be allowed to provision PDs in zones where nodes exist`,
+
+			// Tries to ssh directly to a node. None of our nodes have external IPs
+			`[k8s.io] [sig-node] crictl should be able to run crictl on the node`,
```

Comment:
we should add `\[sig-storage\] Flexvolumes should be mountable` as they require ssh too...

Reply:
also this probably needs to be `\[k8s.io\] \[sig-node\] crictl should be able to run crictl on the node`